_includes/version-warning.html

```diff
@@ -27,18 +27,18 @@
     </p>
     <p>
     <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
-    Heptio Ark is now Velero! We’re working on our first release as Velero, as well as instructions
-    for migrating your Ark deployments and data to Velero. Stay tuned! For more details on the
-    renaming, please read our <a href="https://blogs.vmware.com/cloudnative/2019/02/05/welcoming-heptio-open-source-projects-to-vmware/">blog post</a>.
+    Heptio Ark is now Velero! This v0.11 is our first release as Velero. For instructions on 
```

Comment:
How about "Heptio Ark is now Velero! v0.11 is our first release with the new name. For..."